apps/vapi/Log.cpp

```diff
@@ -0,0 +1,22 @@
+#include "Log.h"
```

Comment:
Do we really need another logging facility? Can we not make use of the existing one that we have, and modify it if needed? It would be preferable to be consistent throughout the codebase.

Reply:
I added this because I believe it is a better fit for a python API than `WASP::MyBase` which works better for a GUI application. I can remove it eventually but I'd prefer to leave it during development as it has features that make debugging easier.